src/core/syscall_request.rs

```diff
@@ -16,6 +17,15 @@ pub(crate) struct EmitEventStruct {
     pub(crate) data: Relocatable,
 }
 
+#[allow(unused)] // TODO: Remove once used.
+pub(crate) struct LibraryCallStruct {
+    pub(crate) selector: BigInt,
+    pub(crate) class_hash: BigInt,
+    pub(crate) function_selector: BigInt,
+    pub(crate) calldata_size: BigInt,
```

Comment:
This is most likely a `usize`. How much data can you fit in memory at a time? Never more than your address space.

Reply:
selector has to be a BigInt, the rest of the parameters can be `usize`. With the last merge, the function `get_integer` already fetches the data as usize, so changes should only be done on data types declarations.